code/datums/trading/unique.dm

```diff
@@ -1,7 +1,7 @@
 /datum/trader/ship/unique
 	trade_flags = TRADER_WANTED_ONLY|TRADER_GOODS
-	want_multiplier = 5
-	typical_duration = 10
+	want_multiplier = 100
+	typical_duration = 20
```

Comment:
Also a no on that.
If you want them to stay longer you can pay them.

Reply:
Code updated to reflect suggested change, resolving.